### PR TITLE
Small fix for secret doc

### PR DESCRIPTION
**Pull request by deyuanix**

### CODE REVIEW

File: docs/secrets.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ The pods:
     }
   }
 },
-{
+ {
```
Comment (lavalamp) [inline docs/secrets.md]: extra space

**Comment by lavalamp** [comment]
LGTM with one fix

File: docs/secrets.md [hunk 2]
```diff
@@ -272,7 +272,7 @@ Note how the specs for the two pods differ only in one field;  this facilitates
 creating pods with different capabilities from a common pod config template.
 
 ### Use-case: Secret visible to one container in a pod
-<a name="use-case-two-containers"></a> 
+<a name="use-case-two-containers"></a>
```
Comment (ddysher) [inline docs/secrets.md]: Hmm, json formatter tells me there needs to be a space.  Removed.

File: docs/secrets.md (hunk 1 above)
Comment (ddysher) [inline docs/secrets.md]: Hmm, json formatter tells me there needs to be a space. Removed.

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**